lnclient/ldk/ldk.go

```diff
@@ -1012,11 +1012,39 @@ func (ls *LDKService) GetNodeConnectionInfo(ctx context.Context) (nodeConnection
 		return nil, err
 	}*/
 
-	return &lnclient.NodeConnectionInfo{
+	nodeConnectionInfo = &lnclient.NodeConnectionInfo{
 		Pubkey: ls.node.NodeId(),
-		// Address: parts[0],
-		// Port:    port,
-	}, nil
+	}
+
+	// If NODE_IPV6 is set, include the connection address and port
```

Comment:
@frnandu I actually don't think we should do it this way,

the `LDKListeningAddresses` and `LDKAnnouncementAddresses` should be enough already and not require a new `NODE_IPV6` field